senders/client_factory.go

```diff
@@ -174,3 +183,10 @@ func TracesPort(port int) Option {
 		cfg.TracesPort = port
 	}
 }
+
+// SDKMetricsTags sets internal SDK metrics.
+func SDKMetricsTags(tags map[string]string) Option {
+	return func(cfg *configuration) {
+		cfg.SDKMetricsTags = tags
```

Comment:
Just to be safe, consider making a defensive copy of tags here.  That way the Option SDKMetricsTags() creates won't change even if the caller changes the map they passed in later on.